src/dswx_sar/initial_threshold.py

```diff
@@ -1005,9 +1005,9 @@ def determine_threshold(
                                   bounds=((-35, 0, 0.01,
                                            -35, 0, 0.01,
                                            -35, 0, 0.01),
```

Comment:
I think that was suggested in our SciPy Issue [here](https://github.com/scipy/scipy/issues/20686), but what do you think about the bounds of the standard deviation from `0` to a very small value like `0.01`?

Reply:
I'm curious about what happens in using the zero values. Anyway, I'll change the value to 0.001 if it is safer.